src/QsCompiler/LanguageServer/Utils.cs

```diff
@@ -123,4 +143,42 @@ public override void Initialize(IEventSource eventSource)
             }
         }
     }
+
+    internal static class DotNetSdkHelper
+    {
+        private static readonly Regex DotNet31Regex = new Regex(@"^3\.1\.\d+", RegexOptions.Multiline | RegexOptions.Compiled);
+        private static readonly Regex DotNet50Regex = new Regex(@"^5\.0\.\d+", RegexOptions.Multiline | RegexOptions.Compiled);
+
+        public class DotNetSDKVersions
+        {
+            public bool HasDotNet50 { get; set; }
+
+            public bool HasDotNet31 { get; set; }
```

Comment:
It looks like we are only using `HasDotNet31`. Can this class simply be a function that returns true if .NET Core SDK 3.1 is installed?

Reply:
Done